SeriesMaster-Frontend/App/components/AppInputText.js

```diff
@@ -1,13 +1,17 @@
 import React from "react";
 import styled from "styled-components/native";
 import PropTypes from "prop-types";
+import { MaterialCommunityIcons } from "@expo/vector-icons";
+import colors from "../config/colors";
 
 const InputContainer = styled.View`
   background: #fcfcfc;
```

Comment:
You are using theming on all colors, why not here?

Reply:
good point!